src/kia_hyundai_api/us_kia.py

```diff
@@ -291,11 +340,17 @@ async def start_climate(
                     "unit": 1,
                     "value": str(set_temp),
                 },
+                "heatVentSeat": {
+                    "driverSeat": self._seat_settings(driver_seat),
+                    "passengerSeat": self._seat_settings(passenger_seat),
+                    "rearLeftSeat": self._seat_settings(left_rear_seat),
+                    "rearRightSeat": self._seat_settings(right_rear_seat)
+                },
                 "defrost": defrost,
                 "heatingAccessory": {
                     "rearWindow": int(heating),
                     "sideMirror": int(heating),
-                    "steeringWheel": int(heating),
+                    "steeringWheel": int(heating)
```

Comment:
please leave the trailing comma so the next addition doesn't have to change a second line when adding another one.

Reply:
👍 the formatter has a mind of its own sometimes.